modules/3d-tiles/package.json

```diff
@@ -29,7 +29,7 @@
     "README.md"
   ],
   "scripts": {
-    "pre-build": "npm run build-bundle && npm run build-bundle -- --env.dev",
+    "pre-build": "npm run build-bundle && npm run build-bundle -- --env.dev && npm run build-bundle -- --env.es5",
```

Comment:
We should investigate updating the webpack script to return an array of configurations so we don't need to update the script in all places.

Reply:
Done